### PR TITLE
Jb54310

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
The very first commit belongs to another PR

**Comment by martyone** [comment]
Initially I tried to remove the need to connectTo prior to lockDown, but that would not be a trivial extension to the state machine.

File: src/tools/sfdk/command.cpp [hunk 1]
```diff
@@ -1674,7 +1674,7 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
         if (!P::checkPositionalArgumentsCount(parser.positionalArguments(), 1, 1))
             return BadUsage;
 
-        const QString name = positionalArguments.first();
+        const QString name = parser.positionalArguments().first();
```
Comment (vigejolla) [inline src/tools/sfdk/command.cpp:1677]: How on earth did this end up in the commit :-o talking about the non-compiling code of course

**Review by vigejolla — APPROVED** [review]
LGTM. Please file a bug about checking if the configuration has changed, unless you have already done so.

**Pull request merged**